Source/WTF/wtf/text/CString.cpp

```diff
@@ -45,31 +45,31 @@ Ref<CStringBuffer> CStringBuffer::createUninitialized(size_t length)
     return adoptRef(*new (NotNull, stringBuffer) CStringBuffer(length));
 }
 
-CString::CString(const char* str)
+CString::CString(const char* string)
 {
-    if (!str)
+    if (!string)
         return;
 
-    init(str, strlen(str));
+    init(WTF::span(string));
 }
 
-CString::CString(const char* str, size_t length)
+CString::CString(std::span<const char> string)
 {
-    if (!str) {
-        ASSERT(!length);
+    if (!string.data()) {
+        ASSERT(string.empty());
         return;
     }
 
-    init(str, length);
+    init(string);
 }
 
-void CString::init(const char* str, size_t length)
+void CString::init(std::span<const char> string)
```

Comment:
If we made one constructor call the other, we could put this into the constructor instead of having a separate function.

Reply:
Good idea. I'll make the change.